Framework/Algorithms/src/CalMuonDetectorPhases.cpp

```diff
@@ -15,6 +15,13 @@
 #include "MantidAPI/WorkspaceFactory.h"
 #include "MantidAPI/WorkspaceGroup.h"
 
+namespace {
+bool isZero(double value) {
+  bool result = value == 0;
```

Comment:
Why the unnecessary variable initialisation? Would it not be more efficient (and possibly more readable) to just do `return value == 0`? The same goes for `PhaseQuadMuon` and `RemoveExpDecay`

Reply:
thats left over from debugging. I will update them